docs/configuration/server.md

```diff
@@ -70,6 +78,11 @@ the client and the server by encrypting data.
     certificate (via wget `--certificate` and `--private-key`) to successfully authenticate with the
     server. For `want` mode, client certificates are optional for health checks.
 
+* **Reverse Proxy**:
+  * If the agent runs behind a reverse proxy `forward-headers-strategy: framework` is necessary for 
+    the correct construction of links. 
+
+
 ## References
 
 * **Spring Boot Documentation**:
```

Comment:
Add Link to https://docs.spring.io/spring-boot/how-to/webserver.html#howto.webserver.use-behind-a-proxy-server